### DISCUSSION TITLE
Don´t know my Index for the migration to Firefly

### DISCUSSION THREAD

**Discussion by Bennofant** [iotaledger/firefly#3120] [Help & Support]
Hello Community,

I ve a problem by migrating my old Iotas from Trinity to Firefly.
I had two wallets on trinity, with two different Ledger Nano S.
My main wallet was the Ledger-Index 0 but I dont know the second one.
Is there another possibility to find out which number my second Ledger-Index had, or do I need to try until I reach 2147483647, thats insane.

Thank you in advance,

Ben

**Comment by MiKeF05** [comment]
Hello Ben,

if you have installed Trinity you can log in and check your ledger index. Make sure to use the last released version. Before you open Trinity make sure to disable your internet connection. Otherwise you will get the info that Trinity has been discontinued.

After you are logged in you need to select your wallet and go to "View Seed". There should be displayed your ledger index.
